packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -280,9 +281,30 @@ class EditableText extends StatefulWidget {
   /// The type of keyboard to use for editing the text.
   final TextInputType keyboardType;
```

Comment:
somewhere (maybe the class documentation) we should have a section that talks about the event lifecycle (e.g. that onEditingComplete happens before onSubmitted but after onChanged, or whatever)

Reply:
I added a few lines to the class docs.  Let me know if you have any comments.